### PR TITLE
HSC-207: Remove Examination field from the Physical Examination form

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket [here](https://mekomsolutions.atlassian.net/browse/HSC-207)

### CODE REVIEW

**Comment by kazlaw** [comment]
image


@Ruhanga, please have a look

File: configuration/concepts/void_examinations.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1600
+18e738c6-a8b9-4f42-a87f-a2d7f291aca3,TRUE,,Examination,Examen,Examination,Examen,,,Misc,Coded,,Normal;Anormal,,,,,,,,,
```
Comment (Ruhanga) [inline configuration/concepts/void_examinations.csv]: I guess you just need the `uuid` and `void/retire` fields.

**Review by Ruhanga — COMMENTED** [review]
Just one little comment. Otherwise generally LGTM.

File: configuration/concepts/void_examinations.csv (hunk 1 above)
Comment (kazlaw) [reply configuration/concepts/void_examinations.csv]: @Ruhanga do I remove the other fields?

Comment (kazlaw) [reply configuration/concepts/void_examinations.csv]: To something like below ?

`18e738c6-a8b9-4f42-a87f-a2d7f291aca3,TRUE,,,,,,,,,Coded,,,,,,,,,,,`

Comment (Ruhanga) [reply configuration/concepts/void_examinations.csv]: > the `uuid` and `void/retire` fields.

Does it not work, the suggestion above, something along the lines 

`18e738c6-a8b9-4f42-a87f-a2d7f291aca3,TRUE,,,,,,,,,,,,,,,,,,,,,,,,,,,......`?

**Review by Ruhanga — APPROVED** [review]
LGTM.

**Pull request merged**